src/com/visualization/piechart/piechart.js

```diff
@@ -23,12 +23,12 @@ export default class PieChart extends Component {
 
     render( props ) {
 
-        if ( !(props && props['lables'] && props['values']) ) {
+        if ( !(props && props['labels'] && props['values']) ) {
```

Comment:
why not `props.labels?

Reply:
I cannot be certain that someone has passed in labels={} to the component.

Reply:
`props.label` should still evaluate as false in that case.

Reply:
Yeah this is one of those JS silly things. As long as we're sure the parent (props) is an object, you can safely use `props.labels` and it wont die a horrible death.

Reply:
JS is weird, changed.